packages/cli/uni-builder/src/shared/utils.ts

```diff
@@ -0,0 +1,59 @@
+import type {
+  RsbuildTarget,
+  NormalizedConfig,
+  FilenameConfig,
+} from '@rsbuild/core';
+
+export const RUNTIME_CHUNK_NAME = 'builder-runtime';
+
+export const TS_REGEX = /\.(?:ts|mts|cts|tsx)$/;
+
+export function isServerTarget(target: RsbuildTarget[]) {
+  return (Array.isArray(target) ? target : [target]).some(item =>
+    ['node', 'service-worker'].includes(item),
+  );
+}
+
+export function getFilename(
+  config: NormalizedConfig,
+  type: 'js',
+  isProd: boolean,
+): NonNullable<FilenameConfig['js']>;
+export function getFilename(
+  config: NormalizedConfig,
+  type: Exclude<keyof FilenameConfig, 'js'>,
+  isProd: boolean,
+): string;
+export function getFilename(
```

Comment:
Only `getFilename(rsbuildConfig, 'media')` is used, do we need to implement a complete `getFilename` function?